Server/Controllers/MatchController.cs

```diff
@@ -84,8 +89,11 @@ public async Task<BeforeMatchResponse> Before(BeforeMatchRequest request)
             var mapName = request.MapName;
 
             var realSteamIds = request.Players.Select(ulong.Parse).ToList();
+            // We need to do another call in order to get the top players for leaderBoard
+            List<LeaderBoardPlayer> topPlayers = await GetTopPlayers();
```

Comment:
```suggestion
            var topPlayers = await GetTopPlayers();
```

I probably should add these conventions to editorconfig, but basically always use `var`